rpm_spec/subpackages/manageiq-gemset

```diff
@@ -46,6 +46,18 @@ Requires: network-scripts
 %description gemset
 %{product_summary} Gemset
 
+%post gemset
+# Remove any old systemd unit files
+files="amazon_agent_coordinator amazon_cloud_manager_event_catcher amazon_cloud_manager_metrics_collector"
```

Comment:
TODO this obviously isn't the complete list :laughing: 
Thinking I can loop through the provider types then worker types to not have to check each one.

Just a simple `rm -rf /etc/systemd/system/*_refresh_worker*` felt too dangerous :)